web/src/containers/Notification.js

```diff
@@ -7,12 +7,16 @@ import { processQueue, closeNotification } from '../actions/notification';
 
 class Notification extends Component {
   handleClose = (_, reason) => {
-    if (reason === 'clickaway') return;
-    this.props.closeNotification();
+    if (reason === 'clickaway') {
+      return;
+    }
+    const { closeNotification: action } = this.props;
+    action();
```

Comment:
I don't quite understand this one. Why is it `action()` instead of `closeNotification()`?

Reply:
A variable named `closeNotification` exists in an upper scope and eslint will complain about that, so I renamed it when I destructured it out of props.  I went with `action` because it represents a redux action and I hoped that'd make it clear what it is, but I reckon it did not.  😬